src/Components/ApiFetch/ApodApiFetch.tsx

```diff
@@ -0,0 +1,24 @@
+﻿import React from "react";
+
+
+
+export function parseApiData() {
+    var requestOptions: any = {
+        method: 'GET',
+        redirect: 'follow'
+    };
+
+    var apiUrl: string = "https://api.nasa.gov/planetary/apod?api_key="+process.env.REACT_APP_NASA_API_KEY;
+    let parsedResult:JSON = JSON;
+    fetch(apiUrl, requestOptions)
+        .then(response => response.text())
+        .then(result => {
+           parsedResult = JSON.parse(result);
+           console.log(parsedResult);
+        }
+            )
+        .catch(error => console.log('error', error));
+    
+    return parsedResult;
```

Comment:
This won't work sadly.
`parsedResult` will always just be `undefined` because the fetch is `async`.

We'll need to add some `await`s.